jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/RangeSlider.java

```diff
@@ -22,57 +23,60 @@ public class RangeSlider extends UIBaseElement<RangeSliderAssert> {
+    private static final String THUMB_LABEL_CONTAINER_LOCATOR = ".v-slider__thumb-label-container";
+    private static final String THUMB_LABEL_LOCATOR = ".v-slider__thumb-label";
+    private static final String TRACK_CONTAINER_LOCATOR = ".v-slider__track-container";
+    private static final String TRACK_BACKGROUND_LOCATOR = ".v-slider__track-background";
+    private static final String TRACK_FILL_LOCATOR = ".v-slider__track-fill";
+    private static final String TICKS_CONTAINER_LOCATOR = ".v-slider__ticks-container";
+    private static final String ALWAYS_SHOW = "v-slider__ticks-container--always-show";
+    private static final String TICK_LABEL_LOCATOR = ".v-slider__tick-label";
 
     @JDIAction("Get track container from '{name}'")
     protected UIElement getTrackContainer() {
-        return $(trackContainerLocator, this);
+        return $(TRACK_CONTAINER_LOCATOR, this);
     }
 
     @JDIAction("Get track background from '{name}'")
     public WebList getBackground() {
-        return $$(trackBackgroundLocator, getTrackContainer());
+        return $$(TRACK_BACKGROUND_LOCATOR, getTrackContainer());
     }
 
     @JDIAction("Get thumb container from '{name}'")
     protected WebList getThumbContainer() {
-        return $$(thumbContainerLocator, this);
+        return $$(THUMB_CONTAINER_LOCATOR, this);
     }
 
     @JDIAction("Get left thumb from '{name}'")
     public UIElement getLeftThumb() {
-        return $(thumbLocator, getThumbContainer().get(1));
+        return $(THUMB_LOCATOR, getThumbContainer().get(1));
     }
 
     @JDIAction("Get right thumb from '{name}'")
     public UIElement getRightThumb() {
-        return $(thumbLocator, getThumbContainer().get(2));
+        return $(THUMB_LOCATOR, getThumbContainer().get(2));
     }
 
     @JDIAction("Get thumb label from '{name}'")
-    public WebList getThumbLabel() {
-        return $$(thumbLabelLocator, this);
+    public List<Label> getThumbLabels() {
+        WebList webList = $$(THUMB_LABEL_LOCATOR, this);
+        return webList.stream().map(element -> new Label().setCore(Label.class, element)).collect(Collectors.toList());
+    }
+
+    @JDIAction("Check that '{name}' thumb label is exist")
+    public boolean hasThumbLabels() {
+       return getThumbLabels().stream().map(ICoreElement::isExist).collect(Collectors.toList()).isEmpty();
     }
 
     @JDIAction("Get tick label value from '{name}'")
     public String getTickLabel(int index) {
-        return $$(tickLabelLocator, this).get(index).getValue();
+        return $$(TICK_LABEL_LOCATOR, this).get(index).getValue();
     }
 
     @JDIAction("Get track fill container from '{name}'")
-    protected UIElement getTrackFillContainer() {return $(trackFillLocator, getTrackContainer());}
+    protected UIElement getTrackFillContainer() {return $(TRACK_FILL_LOCATOR, getTrackContainer());}
 
     @JDIAction("Get value from '{name}'")
     public List<Integer> getValue() {
```

Comment:
We have a rule that we don't use get at the beginning of a method. Please check it elsewhere too
![image](https://user-images.githubusercontent.com/74856431/214813205-1a4d640a-ebc8-495d-a08d-9e3d2f6a63ba.png)
https://jdi-family.atlassian.net/wiki/spaces/JDIL/pages/47349767/Pages+and+elements+management\